patterns/core/sass/_plugins.scss

```diff
@@ -3,3 +3,5 @@
 //////////////////////////////
 @import '../../../bower_components/sass-toolkit/stylesheets/toolkit';
 @import '../../../bower_components/ibm-colors/ibm-colors';
```

Comment:
Can't interpolate import URLs, so we can't DRY this out until we can move to eyeglass